persistence/sql/persister.go

```diff
@@ -135,6 +136,57 @@ type node interface {
 	GetNID() uuid.UUID
 }
 
+func (p *Persister) CleanupDatabase(ctx context.Context, wait time.Duration) error {
+	currentTime := time.Now()
+	deleteLimit := p.r.Config(ctx).DatabaseCleanupBatchSize()
+	p.r.Logger().Printf("Cleaning up first %d records older than %s\n", deleteLimit, currentTime)
```

Comment:
I went with the `--keep-last` flag. I'm not sure if the tables flag is needed. We would run multiple crons and drop records without being sure that they don't fire at the same time. Doing tables one after another with a proper sleep value mitigates replication lag issues

Reply:
The `--tables` flag was just a random idea that came to mind. I think it is fine as is, in case someone needs per-table cleanup times we can still add that later.